src/bindings/jna/README.md

```diff
@@ -17,27 +20,48 @@ then you can compile and run [HelloElektra](HelloElektra.java):
 
     javac HelloElektra.java && java HelloElektra
 
-For plugin development, see [plugins](elektra/plugin)
+Alternatively you can also specify the classpath directly, both during compilation and execution. Note that in this example, we have the elektra and jna libraries in different directories, as mac doesn't allow writing to /usr/share/java in never versions. Also note its important in that case to include this directory containing the compiled HelloElektra.class when executing it, otherwise it will not be found:
+
+	javac -cp ./libelektra4j.jar:./libelektra4j/jna.jar HelloElektra.java
+	java -cp .:./libelektra4j.jar:./libelektra4j/jna.jar HelloElektra
+
+For plugin development, see [plugins](libelektra4j/plugin)
 and also [here](/src/plugins/jni) for more information.
 
+### Maven ###
+
+In case you want to use maven as your build system, simply go into the libelektra4j subfolder and execute mvn install. This will install the plugin into your local maven repository. Afterwards you can include it as a dependency into your maven project, and given that libelektra is actually installed on your system you can use it by including the maven dependency:
+	<groupId>org.libelektra</groupId>
+	<artifactId>libelektra4j</artifactId>
+	<version>0.0.1-SNAPSHOT</version>
```

Comment:
From my point of view, the main issue is the whole build process.

Now we've two different ways to build the jna-bindings jar, with two different outcomes:

- the cmake way
- and the maven way

cmake builds a plain jar, containing only the compiled byte code / class files and maven, building a 'maven' aware jar file, containing the pom in addition.

I think this is one build process too much. Which one to use when? Are the two artefacts really the same? I would prefer the maven way, as this is currently the default within the Java world. Also an upload to maven central would be very nice. 
Suppose you have installed elektra already, e.g. through apt-get, yum or similar. Then a Java developer simply can start by adding the 'libelektra4j' dependency into his pom.xml and run 'mvn install'.

Also, currently the Unit tests aren't integrated in the cmake build process (as far as I've seen). So basically wrapping the 'mvn' calls in cmake would be the best solution from my point of view.

Then we should definitely use the elektra version as pom version. So an update to the jna bindings issues a new elektra version, whereas a new elektra version issues a new jna-bindings version.

What do you think?
@e1528532 @markus2330